pyproject.toml

```
@@ -1,3 +1,25 @@
+[build-system]
+requires = ["hatchling"]
+build-backend = "hatchling.build"
+
+[project]
+name = "kyber-py"
+version = "1.0.1"
+requires-python = ">=3.9"
```

nitpick: I think a sparse look improves readability.
```suggestion
requires-python = ">= 3.9"
```